src/System.Linq.Expressions/tests/BinaryOperators/Bitwise/BinaryExclusiveOrTests.cs

```diff
@@ -273,7 +273,7 @@ public static void ToStringTest()
             BinaryExpression e = Expression.ExclusiveOr(Expression.Parameter(typeof(int), "a"), Expression.Parameter(typeof(int), "b"));
             Assert.Equal("(a ^ b)", e.ToString());
 
-            // NB: Unlike And and Or, there's no special case for bool and bool? here.
+            // NB: Unlike AND and OR, there's no special case for bool and bool? here.
```

Comment:
💭 Is this not talking about [`Expression.And`](https://docs.microsoft.com/en-us/dotnet/api/system.linq.expressions.expression.and) and [`Expression.Or`](https://docs.microsoft.com/en-us/dotnet/api/system.linq.expressions.expression.or)?

```suggestion
            // NB: Unlike 'Expression.And' and 'Expression.Or', there's no special case for bool and bool? here.
```